### PR TITLE
OHRM5X-2062: P2 Defect fixes

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: src/client/src/orangehrmAdminPlugin/pages/employmentStatus/EditEmploymentStatus.vue [hunk 1]
```diff
@@ -101,12 +101,15 @@ export default {
         this.employmentStatus.id = data.id;
         this.employmentStatus.name = data.name;
         // Fetch list data for unique test
-        return this.http.getAll();
+        return this.http.getAll({limit: 0});
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/employmentStatus/EditEmploymentStatus.vue:104]: TODO:: Need to check for remote validation generic API endpoint

File: src/client/src/orangehrmAdminPlugin/pages/jobTitle/SaveJobTitle.vue [hunk 2]
```diff
@@ -149,7 +149,10 @@ export default {
       .then((response) => {
         const {data} = response.data;
         this.rules.title.push((v) => {
-          const index = data.findIndex((item) => item.title == v);
+          const index = data.findIndex(
+            (item) =>
+              String(item.title).toLowerCase() == String(v).toLowerCase(),
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/jobTitle/SaveJobTitle.vue:154]: TODO:: move to remote validation

File: src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue [hunk 3]
```diff
@@ -115,10 +115,14 @@ export default {
   methods: {
     onSave() {
       this.isLoading = true;
-      this.http.http
-        .put('api/v2/admin/localization', {
-          language: this.configuration.language?.id,
-          dateFormat: this.configuration.dateFormat?.id,
+      this.http
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue:118]: any special reason for this?

File: src/client/src/orangehrmPimPlugin/pages/employee/SaveEmployee.vue [hunk 4]
```diff
@@ -290,12 +293,16 @@ export default {
...
           }
           if (this.createLogin && data?.data) {
-            return this.http.http.post('api/v2/admin/users', {
-              username: this.user.username,
-              password: this.user.password,
-              status: this.user.status == '1',
-              userRoleId: this.user.userRoleId,
-              empNumber: data.data.empNumber,
+            return this.http.request({
+              type: 'GET',
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPimPlugin/pages/employee/SaveEmployee.vue:297]:
```suggestion
              type: 'POST',
```
:thinking:

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue (hunk 3 above)
Comment (Super-Chama) [reply src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue:118]: refactored this code to use `request` method instance of accessing internal axios from api service instance

File: src/client/src/orangehrmPimPlugin/pages/employee/SaveEmployee.vue (hunk 4 above)
Comment (Super-Chama) [reply src/client/src/orangehrmPimPlugin/pages/employee/SaveEmployee.vue]: ops, good save. thanks

File: src/client/src/orangehrmPimPlugin/components/ProfileAttachments.vue [hunk 5]
```diff
@@ -298,4 +298,8 @@ export default {
   margin-top: 0;
   margin-bottom: 0;
 }
+.orangehrm-attachment {
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPimPlugin/components/ProfileAttachments.vue:301]: Fix for #1584

**Pull request merged**